client/README.md

```diff
@@ -35,7 +35,47 @@ You can then use the following command to start the server on your machine:
 npm start
 ```
 
-### Usage
+### StoryBook Organization
```

Comment:
Can you maybe provide some more narrative? Think of like writing an email to someone explaining what this is. You may need a short paragraph here explaining what this section is about. Then a heading for each section of the component library and what goes into those sections and why.

Reply:
Because there were so many headings i felt like using bold to represent the folders might be easier to read and not cause confusion that its a different section in the readme. I can change it back however if youd like.